src/include/storage/storage_defs.h

```diff
@@ -263,8 +263,12 @@ class VarlenEntry {
     TERRIER_ASSERT(size <= InlineThreshold(), "varlen value must be small enough for inlining to happen");
```

Comment:
@mbutrovich Could you take a look at just the changes in this file? 

1. Prefix is now always zeroed.
2. Inlined strings use CRC hashing, longer strings use XXH3 hashing.

Reply:
Those parts LGTM.